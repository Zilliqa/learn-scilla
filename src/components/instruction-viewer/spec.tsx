import React from 'react';
import InstructionViewer from '.';
import * as renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(<InstructionViewer instruction={`## markdown`} />).toJSON();
  expect(tree).toMatchSnapshot();
});
