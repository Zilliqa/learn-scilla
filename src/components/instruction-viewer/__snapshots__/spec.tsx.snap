// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="code-instruction-container"
>
  <h2>
    markdown
  </h2>
</div>
`;
